posts/2023-10-23-run-linux-windows-on-mac.adoc

```diff
@@ -0,0 +1,168 @@
+
+You’ll need to specify the amount of memory and the number of CPU cores that the VM should use.
+You can start with the default settings and adjust them later if needed.
+
+.Choose Hardware
+image::utm-choose-hardware.png[Choose Hardware, 600]
+
+Next, you’ll encounter the `Shared directory` screen. This is essential if you wish to be able to transfer files
+between your Mac and the VM. Click `Browse…` to select an existing folder or create a new one for this purpose.
+
+.Shared Directory
+image::utm-shared-directory.png[Shared Directory, 600]
+
+The final `Summary` screen provides an overview of your chosen settings and allows you to name the VM.
+
+.Summary
+image::utm-summary-screen.png[Summary, 600]
+
+Once you click `Save`, UTM will begin preparing the VM. Upon completion, your new Windows VM will be listed.
+
+.List of created images
+image::utm-image-ready.png[Summary, 600]
+
+You’re now ready to boot the Windows VM. Click on the large arrow to initiate the VM. Within a few minutes,
+you should see the Windows setup screen. If you encounter a hurdle at the `Let's connect you to a network` screen during
+the Windows setup, shut down the VM and restart it. When you reach the
+`Language selection` screen, press `Shift-F10` to launch the `Windows command prompt`. Type `OOBE\BYPASSNRO` and press Enter.
+This command instructs Windows to bypass the network setup for the time being.
+
+.Configure Windows to allow skipping network setup
+image::utm-configure-no-network.png[Summary, 600]
+
+The VM will then restart. During the Windows setup procedure, you’ll be presented with an option for `I don’t have internet`
+when you arrive to the network screen.
+
+.Configure Network
+image::utm-choose-no-internet.png[No internet, 600]
+
+Select `I don’t have internet` and proceed to complete the Windows setup. We’ll address the internet connection issue later.
+
+Once you’ve completed the setup, you should be able to see the Windows desktop.
+
+.Windows Desktop
+image::utm-windows-startscreen.png[Windows Desktop, 600]
+
+Now, let’s address the internet connection issue. UTM has already set up a drive within the Windows VM to rectify this problem.
+Navigate to `My Computer` > `CD Drive (D:) UTM` and launch `spice-guest-tools-xxx.exe` where xxx is the version number.
+Follow the on-screen instructions to install these tools using the standard Windows installer. Once the installation is complete, reboot the VM.
+
+.SPICE Tools available in Drive D
+image::utm-install-spice-tools.png[Install spice tools, 600]
+
+You should now be able to enjoy your Windows VM with an active internet connection.
+
+.Windows VM with internet
+image::utm-windows-with-internet.png[Windows with internet, 600]
+
+
+== Conclusion
+In this blog post, we successfully set up both a Linux and a Windows VM. The Linux VM was configured
+using XCode, the ‘RunningGUILinuxInAVirtualMachineOnAMac’ project, and a free Linux distribution, requiring minimal
```

Comment:
My suggestion is to tweak this to: The Linux VM was configured using XCode, the ‘RunningGUILinuxInAVirtualMachineOnAMac’ project, and a free Linux distribution--all requiring minimal configuration.